src/GitHub.Exports/Services/IGitService.cs

```diff
@@ -7,6 +7,19 @@ namespace GitHub.Services
 {
     public interface IGitService
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LocalRepositoryModel"/> class.
+        /// </summary>
+        /// <param name="localPath">The repository's local path.</param>
+        /// <returns>A repository model.</returns>
+        ILocalRepositoryModel CreateLocalRepositoryModel(string localPath);
+
+        /// <summary>
+        /// Updates the ClonePath information based on the local path
```

Comment:
The `CurrentBranch` property no longer exists. I've renamed this method to `RefreshCloneUrl` to be explicit about what it does. I'm hoping it will go away completely in a future refactor.